manifests/init.pp

```diff
@@ -112,11 +110,11 @@
 
   contain letsencrypt::renew
 
-  # TODO: do we need this command when installing from package?
   exec { 'initialize letsencrypt':
```

Comment:
I think this only works with the VCS checkout. Perhaps it should be hidden behind `if $install_method == 'vcs' {` as well?

Reply:
Updated